metrics/metrics-core/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>slf4j-log4j12</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
i think spring-core already in the dependency tree.

Reply:
Done